rwengine/src/script/ScriptMachine.cpp

```diff
@@ -148,10 +148,15 @@ void ScriptMachine::executeThread(SCMThread &t, int msPassed)
 #if RW_SCRIPT_DEBUG
 		if (strcmp(t.name, "EIGHT") == 0)
```

Comment:
Can we get rid of this or move the thread name somewhere else where it's easier to change it?